python/lsst/obs/comCam/ingest.py

```diff
@@ -34,23 +50,57 @@ def getInfo(self, filename):
         if runId != phuInfo['run']:
             raise RuntimeError("Expected runId %s, found %s from path %s" % phuInfo['run'], runId, pathname)
 
-        phuInfo['raftId'] = raftId
-        phuInfo['field'] = acquisitionType
-        phuInfo['jobId'] = int(jobId)
-        phuInfo['raft'] = 'R00'
-        phuInfo['ccd'] = sensorLocationInRaft
+        phuInfo['raftId'] = raftId # also in the header - RAFTNAME
+        phuInfo['field'] = acquisitionType # NOT in the header
+        phuInfo['jobId'] = int(jobId) #  NOT in the header, but do we need it?
```

Comment:
Have you determined whether we need the "do we need it"s? If we don't, drop them, if you have reason to believe they'll be useful in the future, just remove the comments.